hs_access_control/models/user.py

```diff
@@ -2274,7 +2280,9 @@ def can_undo_share_resource_with_group(self, this_resource, this_group):
         if not self.user.is_active:
             raise PermissionDenied("Requesting user is not active")
         if not this_group.gaccess.active:
-            raise PermissionDenied("Group is not active")
+            # This causes problems with display of resources
+            # raise PermissionDenied("Group is not active")
+            return False
```

Comment:
Similar comment as above: @alvacouch Should line 2281 be changed to return False rather than raise PermissionDenied exception as well?

Reply:
Same reasoning. Exception is warranted for self.user.